electron/main.js

```diff
@@ -1,35 +1,21 @@
 'use strict';
 
-const app = require('app');
-const BrowserWindow = require('browser-window');
-const ipc = require('electron').ipcMain;
-const notifier = require('node-notifier');
+const electron = require('electron');
+const app = electron.app;
+const BrowserWindow = electron.BrowserWindow;
```

Comment:
'const' is available in ES6 (use esnext option) or Mozilla JS extensions (use moz).